met-web/src/components/publicDashboard/SubmissionTrend/SubmissionTrend.tsx

```diff
@@ -73,6 +81,10 @@ const SubmissionTrend = ({ engagement, engagementIsLoading }: SubmissionTrendPro
         } catch (error) {
             console.log(error);
             setIsError(true);
+            if (axios.isAxiosError(error)) {
+                setErrors(error);
```

Comment:
is there a way without creating a new state noDataError. Maybe if response is 404 don't set error to true and data will be null. If data is null show no data component